spaceship.zsh-theme

```diff
@@ -234,6 +256,7 @@ PROMPT=''
 [[ $SPACESHIP_PROMPT_ADD_NEWLINE == true ]] && PROMPT="$PROMPT$NEWLINE"
 PROMPT="$PROMPT"'$(spaceship_build_prompt) '
 [[ $SPACESHIP_PROMPT_SEPARATE_LINE == true ]] && PROMPT="$PROMPT$NEWLINE"
+[[ $SPACESHIP_VI_MODE_SHOW == true ]] && PROMPT="$PROMPT"'$(spaceship_vi_mode) '
```

Comment:
Move this checking into the `spaceship_vi_mode` function.

I am checking `PROMPT_*` params here, because there aren't such functions in theme. Everything else, which are built using functions, should be checked in its own functions.